README.md

```diff
@@ -1 +1,8 @@
 # Heather and Kayla Code Something Together
+
+
+
+
+Demo'd by Kayla Wood and Heather Faerber
```

Comment:
Great sentence! Maybe we should make it say "Demonstrated" to be more professional.